WinUIGallery/Samples/ControlPages/ContentDialogPage.xaml.cs

```diff
@@ -45,4 +45,35 @@ private async void ShowDialog_Click(object sender, RoutedEventArgs e)
             DialogResult.Text = "User cancelled the dialog";
         }
     }
+
+    private async void ShowDialogNoDefault_Click(object sender, RoutedEventArgs e)
+    {
+        ContentDialogExample dialog = new ContentDialogExample();
+
+        // XamlRoot must be set in the case of a ContentDialog running in a Desktop app
+        dialog.XamlRoot = this.XamlRoot;
+        dialog.Style = Application.Current.Resources["DefaultContentDialogStyle"] as Style;
+        dialog.Title = "Delete file?";
+        dialog.PrimaryButtonText = "Delete";
+        dialog.SecondaryButtonText = "Keep";
+        dialog.CloseButtonText = "Cancel";
+        dialog.DefaultButton = ContentDialogButton.None;
+        dialog.Content = new ContentDialogContent();
+        dialog.RequestedTheme = (VisualTreeHelper.GetParent(sender as Button) as StackPanel).ActualTheme;
+
+        var result = await dialog.ShowAsync();
+
+        if (result == ContentDialogResult.Primary)
+        {
+            DialogResultNoDefault.Text = "User deleted the file";
```

Comment:
these text updates need to meet a11y criteria, meaning changes to these should be announced by narrator.

we only need to do it in this file though, do not add that code to the code snippet itself as this is something we need to meet a11y review but do not add any value to the sample code itself

Reply:
See: https://github.com/microsoft/WinUI-Gallery/blob/bd2d3b771bdc90e1dec42828b65abeae0bf35f66/WinUIGallery/Samples/ControlPages/Fundamentals/Controls/ValidatedPasswordBox.cs#L139